packages/config/src/configs/acala.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  AssetMinBuilder,
+  BalanceBuilder,
+  ExtrinsicBuilder,
+  FeeBuilder,
+} from '@moonbeam-network/xcm-builder';
+import { aca, ausd, glmr } from '../assets';
+import { acala, moonbeam } from '../chains';
+import { AssetConfig } from '../types/AssetConfig';
+import { ChainConfig } from '../types/ChainConfig';
+
+export const acalaConfig = new ChainConfig({
+  assets: [
+    new AssetConfig({
+      asset: aca,
+      balance: BalanceBuilder().system().account(),
+      destination: moonbeam,
+      destinationFee: {
+        amount: FeeBuilder().assetManager().assetTypeUnitsPerSecond(),
+        asset: aca,
+      },
+      extrinsic: ExtrinsicBuilder().xTokens().transfer(),
+    }),
+    new AssetConfig({
+      asset: ausd,
+      balance: BalanceBuilder().tokens().accounts(),
+      destination: moonbeam,
+      destinationFee: {
+        amount: FeeBuilder().assetManager().assetTypeUnitsPerSecond(),
+        asset: aca,
```

Comment:
`ausd` transactions are paid by `ausd` itself in the destination, not `aca`, we don't have to send acala along to pay for the fees